### ISSUE TITLE
Boosters dm

### ISSUE THREAD

**Issue by Shlba** [Minehut/Meta#262]
**Platform**
Discord

**Describe the feature**
A message from the minehut bot to the booster when they boost the server, explaining to them their perks as well as how to add someone using <@[ID]>

**How the feature is useful**
Many people do not know how to mention someone using their discord iD. It would also help them understand what their perks are, and how to use them. I am aware that the discord command !boost does a similar job, but it does not include information on how to give a boosters pass to a user

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by RichieNy** [comment]
This seems like a cool idea but, maybe would suggest sending a Direct Message to the user instead of posting it in a channel.

**Comment by Sniped** [comment]
There could just be an info embed in the boosters-info channel.

**Comment by jellz** [comment]
Agreed, we don't need to DM the user when they boost the server. Just having a message in boosters-info explaining the perks of boosting (can probably just be the tag...), and using the channel topic to instruct users on giving out their booster passes (or another info message). I'm against this bot feature.

**Comment by Shlba** [comment]
> Agreed, we don't need to DM the user when they boost the server. Just having a message in boosters-info explaining the perks of boosting (can probably just be the tag...), and using the channel topic to instruct users on giving out their booster passes (or another info message). I'm against this bot feature.

I don't see that as useful because not everyone scrolls to the top, and the #boosters-info channel is clogged with people requesting and removing users. Its just a matter of convenience really, and would be rather useful to remind people on how the system works, any limitations, etc

**Comment by MiginsMC** [comment]
> > Agreed, we don't need to DM the user when they boost the server. Just having a message in boosters-info explaining the perks of boosting (can probably just be the tag...), and using the channel topic to instruct users on giving out their booster passes (or another info message). I'm against this bot feature.
> 
> I don't see that as useful because not everyone scrolls to the top, and the #boosters-info channel is clogged with people requesting and removing users. Its just a matter of convenience really, and would be rather useful to remind people on how the system works, any limitations, etc

Agreed. Although it could just be an automated message that is sent in #boosters such as "Thanks for boosting (name), you can refer to THIS (hyperlink) message for booster information" or something.

**Comment by Sniped** [comment]
> > > Agreed, we don't need to DM the user when they boost the server. Just having a message in boosters-info explaining the perks of boosting (can probably just be the tag...), and using the channel topic to instruct users on giving out their booster passes (or another info message). I'm against this bot feature.
> > 
> > 
> > I don't see that as useful because not everyone scrolls to the top, and the #boosters-info channel is clogged with people requesting and removing users. Its just a matter of convenience really, and would be rather useful to remind people on how the system works, any limitations, etc
> 
> Agreed. Although it could just be an automated message that is sent in #boosters such as "Thanks for boosting (name), you can refer to THIS (hyperlink) message for booster information" or something.

Couldn't the pinned message just be referenced in the channel topic?

**Comment by Shlba** [comment]
> Couldn't the pinned message just be referenced in the channel topic?

The boosters channel is almost always clogged full. A dm doesn't hurt much anyways

**Comment by Sniped** [comment]
In my opinion it seems unwarranted to DM them with an embed, and what does the boosters channel being clogged have to do with channel topic? They would see the message regardless of the activity of the channel itself if I'm interpreting this correctly.
